.gitignore

```diff
@@ -23,8 +23,8 @@ deprecated
 *.sln
 *.local
 stats.html
-.yarn
-.yarnrc.yml
+.yarn/*
+!.yarn/releases
```

Comment:
https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

Reply:
updated